cmip6_downscaling/data/observations.py

```diff
@@ -51,12 +34,11 @@ def open_era5(variables: Union[str, List[str]], start_year: str, end_year: str)
     if isinstance(variables, str):
         variables = [variables]
 
-    stores = [
-        f'https://cmip6downscaling.blob.core.windows.net/cmip6/ERA5_daily/{y}'
-        for y in range(int(start_year), int(end_year) + 1)
-    ]
-    ds = xr.open_mfdataset(stores, engine='zarr', consolidated=True)
```

Comment:
the loss of the `engine='zarr'` flag here is okay since intake handles it, right?

Reply:
yep. That is encoded in the catalog now.